emoji-selector@maestroschan.fr/emojiButton.js

```diff
@@ -54,13 +60,13 @@ var EmojiButton = class EmojiButton {
 
 		// Update the category label on hover, allowing the user to know the
 		// name of the emoji he's copying.
-		this.super_btn.connect('notify::hover', (a, b) => {
-			if (a.hover) {
-				category.super_item.label.text = this.keywords[0];
-			} else {
-				category.super_item.label.text = category.categoryName;
-			}
-		});
+		// this.super_btn.connect('notify::hover', (a, b) => {
+		// 	if (a.hover) {
+		// 		category.super_item.label.text = _( this.keywords[0] );
+		// 	} else {
+		// 		category.super_item.label.text = _( category.categoryName );
+		// 	}
+		// });
```

Comment:
Do this on your system if you prefer, but i'll not merge it here, it's a regression. Seeing the names is essential so people know the emojis' names and could then find them with the search entry

Reply:
(I guess you tried to translate the emojis' names adn keywords, but there are literally thousands of them it's just not realistic)

Reply:
@maoschanz because somebody will search emoji by typing "伞" instead of "umbrella" here I am, I want to translate all KEYWORDs, could it be merged after I finish it?

Reply:
No, people will hover over the umbrella and see that they should search "umbrella" to find it.

You should not waste your time to translate this: dozens of people already translated it, including the Unicode consortium themselves. What i should do is to get their existing data (https://unicode.org/Public/cldr/36/cldr-common-36.0.zip, issue #61 ) and parse the files in the subdirectory "annotations". Downloading files from internet and parsing XML data is something very different from what the extension currently does, so i will need time to properly implement that